recipes/pyramids/LICENSE.md

```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```

Comment:
Is it possible for you to create a new release with the license file included in the source dist? I think you need to add the following `metadata` key in `setup.cfg`
https://github.com/scikit-image/scikit-image/blob/main/setup.cfg#L10